ansible/management-searchengine.yml

```diff
@@ -0,0 +1,253 @@
+# Search ngine + Search Engine Client + Elasticsearch
+
+#- hosts: "{{ idr_environment | default('idr') }}-management-hosts"
+
+ 
+#hosts: localhost
+#connection: local
+
+#- hosts: all
+#  remote_user: root
+#  vars_files:
+#    - /vars/external_vars.yml
+#- hosts: all
+#  vars_files:
+#    file: searchengine_vars.yml
+
+- name: Deploying search engine
+  connection: local
+  hosts: local
+  vars_files:
+    searchengine_vars.yml
```

Comment:
Moving the variables under `group_vars` should allow them to be automatically picked up - see https://docs.ansible.com/ansible/2.8/user_guide/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable

Reply:
I have moved it